test/recipes/82-test_ocsp_cert_chain.t

```diff
@@ -128,10 +128,6 @@ sub run_test {
         if (/^cert_status: ocsp response sent:/) {
             $resp = 1;
             last;
-        } elsif (/^cert_status:/) {
-            ;
-        } else {
-            last;
```

Comment:
I disagree with this change.

What will happen now, if the expected string is not output to stdout, then the test will just keep spinning.  Eventually, I think the CI infrastructure will kill it, but that may take an hour.

I think the change to this file should be reverted.

Reply:
If you insist on changing the test to accommodate the AFL toolchain, then something like this would be better:

```diff
diff --git a/test/recipes/82-test_ocsp_cert_chain.t b/test/recipes/82-test_ocsp_cert_chain.t
index 351ed66b0a..55bcf5f015 100644
--- a/test/recipes/82-test_ocsp_cert_chain.t
+++ b/test/recipes/82-test_ocsp_cert_chain.t
@@ -130,6 +130,8 @@ sub run_test {
             last;
         } elsif (/^cert_status:/) {
             ;
+        } elsif (/^Warning: AFL/) {
+            ;
         } else {
             last;
         }
```

Reply:
Ah, you're right. Except, even without this, it is not perfect because if no connection is established, this will halt forever waiting on server to write any message.

Reply:
However there is a better way to terminate the server if at least one connection is attempted. We should simply use the `-naccept 1` option.